### PR TITLE
Click tracking stop working

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      |
| Related developer documentation PR URL |
| Issue(s) addressed                     |

#### Description:

This is regression issue from https://github.com/mautic/mautic/pull/9265
Clicks from email was not count

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create email with link
3. Try click on link
4. Click was not count
5. After PR should work like before

### CODE REVIEW

**Comment by kuzmany** [comment]
@adiux this looks like is regression issue from your PR https://github.com/mautic/mautic/pull/9265 
Can you check it?

**Review by RCheesley — APPROVED** [review]
Confirmed the issue before applying the PR and that after the PR was applied, the click tracking is working again.

Also confirmed that UTM tags seem to still be getting logged OK.

**Comment by RCheesley** [comment]
@mohit-rocks did you manage to test this in the end?

File: app/bundles/PageBundle/Model/PageModel.php [hunk 1]
```diff
@@ -739,8 +739,10 @@ public function processPageHit(Hit $hit, $page, Request $request, Lead $lead, $t
      */
     public function getHitQuery(Request $request, $page = null)
     {
-        // get all post params
-        $query = $request->request->all();
+        $get  = $request->query->all();
```
Comment (adiux) [inline app/bundles/PageBundle/Model/PageModel.php:742]: @kuzmany what is the difference between this and getQueryFromUrl() on line 752

Comment (kuzmany) [reply app/bundles/PageBundle/Model/PageModel.php:742]: I suppose $query['page_url'] to getQueryFromUrl method is already without ct param in url

**Review by dadarya0 — APPROVED** [review]
It looks good to me

**Pull request merged**